components/tabs/style.ts

```diff
@@ -0,0 +1,48 @@
+import { TabVarient, TabStatus } from '../utils/prop-types'
+import { ZeitUIThemesPalette } from 'components/styles/themes'
+const stack: TabStatus[] = ['disabled', 'active', 'hover', 'default']
+
+export type StatusMap = {
+  [key in TabStatus]?: boolean
+}
+
+export const defaultGetColor = (
+  palette: ZeitUIThemesPalette,
+  varient: TabVarient,
+  status: TabStatus,
+): { color?: string; background: string } => {
+  switch (status) {
```

Comment:
why not use a object like [here](https://github.com/Conflux-Chain/react-ui/blob/9d1141cee430ffb8964d4c38bd608aebaccb1299/components/button/styles.ts#L40)